### PR TITLE
html-forms

### PR DESCRIPTION
**Pull request by easton-morris**
HTMLForms

### CODE REVIEW

**Comment by easton-morris** [comment]
I fixed the typo before submitting, just didn't rerecord lol.

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  <meta charset="UTF-8">
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>
+    Coding Form!
+  </h1>
+  <form action="" method="POST">
+    <div>
+    <label>
```
Comment (BrettAlbright) [inline html-forms/index.html]: you want to make sure that you labels have a `for` attribute which matches the `id` attribute you want to have on the inputs.

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      <p>Email</p>
+      <input type="email" name="email" />
```
Comment (BrettAlbright) [inline html-forms/index.html]: so make sure to have matching `id`s and `for` attributes

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
small updates needed

**Review by BrettAlbright — APPROVED** [review]
👍

**Pull request merged**